Documentation/operations/upgrade.rst

```diff
@@ -444,10 +444,10 @@ has been renamed from ``envoy_cilium_policymap_<node-ip>_<node-id>_`` to ``envoy
 * ``envoy_cilium_policymap_<node-ip>_<node-id>_update_time`` -> ``envoy_cilium_npds_update_time``
 * ``envoy_cilium_policymap_<node-ip>_<node-id>_update_duration`` -> ``envoy_cilium_npds_update_duration``
 * ``envoy_cilium_policymap_<node-ip>_<node-id>_version`` -> ``envoy_cilium_npds_version``
-* ``doublewrite_identity_crd_total_count`` has been renamed to ``doublewrite_identity_crd_total``
-* ``doublewrite_identity_kvstore_total_count`` has been renamed to ``doublewrite_identity_kvstore_total``
-* ``doublewrite_identity_crd_only_count`` has been renamed to ``doublewrite_identity_crd_only_total``
-* ``doublewrite_identity_kvstore_only_count`` has been renamed to ``doublewrite_identity_kvstore_only_total``
+* ``doublewrite_identity_crd_total_count`` has been renamed to ``doublewrite_identity_crd``
```

Comment:
I am not sure about this but I would expect to see:
```
* ``doublewrite_identity_crd_total`` has been renamed to ``doublewrite_identity_crd``
```

Reply:
the previous change in https://github.com/cilium/cilium/pull/36585 was not released yet, so this is kind of the combined update for both PR.